templates/zerver/emails/organization_info.html

```diff
@@ -0,0 +1,25 @@
+{% extends "zerver/emails/compiled/email_base_default.html" %}
+
+{% block illustration %}
+<img src="{{ email_images_base_uri }}/email_logo.png" alt=""/>
+{% endblock %}
+
+{% block content %}
+<p>{{ _('New organization details') }}</p>
+
+<p>
+    {% if realm_creation %}
+        {% trans %} Organization Name : <b>{{ realm_name }}</b>.{% endtrans %}
+    {% endif %}
+</p>
+
+<p>
+    <li>{{ _('Organization URL:') }} {{ realm_name }} <br/></li>
```

Comment:
This should be changed to 2 fields, "Organization name" being the name, and "Organization URL" being `realm_uri`.